Formula/minishift.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
+
+class Minishift < Formula
+  desc "Runs a single-node OpenShift cluster inside a VM"
+  homepage "https://minishift.io/"
+  url "https://github.com/minishift/minishift/archive/v1.1.0.tar.gz"
+  version "1.1.0"
```

Comment:
The version value is used in the tests. Is it still set even if I don't specify it?